packages/nodes-base/nodes/CrateDb/CrateDb.node.ts

```diff
@@ -283,12 +284,14 @@ export class CrateDb implements INodeType {
 			//         executeQuery
 			// ----------------------------------
 
-			const queryResult = await pgQuery(
-				this.getNodeParameter,
+			const queryResult = await pgQueryV2.call(
+				this,
 				pgp,
 				db,
 				items,
 				this.continueOnFail(),
+				undefined,
+				true,
```

Comment:
1. We can improve this pattern of passing in `undefined` and loose booleans by defining an `options` arg.
2. Do we need to explicitly pass in `this.continueOnFail()` if we are already passing in the `this` context?

Reply:
1. Sure, will update,
2. Probably, not but I guess this leaves us the option of not tying the two parameters together, so leaving it as is.